Client/Frontend/Home/RemoteTabsPanel.swift

```diff
@@ -18,9 +18,10 @@ private struct RemoteTabsPanelUX {
     static let RowHeight: CGFloat = SiteTableViewControllerUX.RowHeight
     static let HeaderBackgroundColor = UIColor(rgb: 0xf8f8f8)
 
-    static let EmptyStateTitleFont = UIFont.systemFontOfSize(UIConstants.DeviceFontSize, weight: UIFontWeightMedium)
+    static let EmptyStateTitleFont: UIFont = UIFont.systemFontOfSize(UIConstants.DeviceFontSize, weight: UIFontWeightMedium)
     static let EmptyStateTitleTextColor = UIColor.darkGrayColor()
-    static let EmptyStateInstructionsFont = UIFont.systemFontOfSize(UIConstants.DeviceFontSize - 1, weight: UIFontWeightLight)
+
+    static let EmptyStateInstructionsFont: UIFont = UIFont.systemFontOfSize(UIConstants.DeviceFontSize - 1, weight: UIFontWeightLight)
```

Comment:
Nit: Don't need to declare the type - it can be inferred from UIFont.systemFontOfSize